controllers/doguInstallManager.go

```diff
@@ -209,6 +215,58 @@ func (m *doguInstallManager) createDoguResources(ctx context.Context, doguResour
 	return nil
 }
 
+func (m *doguInstallManager) createConfigs(ctx context.Context, doguName string, logger logr.Logger) (func(error), error) {
+	var doguCfgAlreadyExists, sensitiveCfgAlreadyExists bool
+
+	cleanUp := func(err error) {
```

Comment:
🚫 **[checkstyle]** <contextcheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Function `createConfigs$1` should pass the context parameter
<!-- __reviewdog__:ChBkYzA1NzNiOTcxNDRiMzQ0EgpjaGVja3N0eWxl -->